modules/programs/neovim.nix

```diff
@@ -130,19 +145,68 @@ in
         description = ''
           Generate your init file from your list of plugins and custom commands,
           and loads it from the store via <command>nvim -u /nix/store/hash-vimrc</command>
+
+          </para><para>
+
+          This option is deprecated. Please use the options <varname>extraConfig</varname>
+          and <varname>plugins</varname> which are mutually exclusive with this option.
```

Comment:
@rycee what was the motivation for deprecating the `configure` option?

In shabka, we declare a common [neovim configuration module](https://github.com/kalbasit/shabka/blob/d02d93069fe45e5c4296f56a2ebd1096b7fb1138/modules/neovim/default.nix) that are being used directly for configuring [NixOS](https://github.com/kalbasit/shabka/blob/d02d93069fe45e5c4296f56a2ebd1096b7fb1138/modules/nixos/software/neovim.nix#L50) and [home-manager](https://github.com/kalbasit/shabka/blob/d02d93069fe45e5c4296f56a2ebd1096b7fb1138/modules/home/software/neovim.nix#L51). It would be a shame to lose this consistency.

Reply:
I've opened a separate issue to discuss this: https://github.com/rycee/home-manager/issues/971